tests/assertions_operators.yml

```diff
@@ -0,0 +1,68 @@
+name: Assertions testsuite
+testcases:
+
+  - name: AssertionsOperatorAnd
+    steps:
+    - script: echo 1
+      assertions:
+        - and:
+          - result.systemoutjson ShouldEqual 1 
+          - result.systemoutjson ShouldContain 1
+
+  - name: AssertionsOperatorOr
+    steps:
+    - script: echo 1
+      assertions:
+        - or:
+          - result.systemoutjson ShouldEqual 1 
+          - result.systemoutjson ShouldEqual 2
+
+  - name: AssertionsOperatorOrError
+    steps:
+    - script: echo 1
+      assertions:
+        - or:
+          - result.systemoutjson ShouldEqual 0 
+          - result.systemoutjson ShouldEqual 2
```

Comment:
is there a way to "expect assertions errors" ?
it's kind of meta but the aim here would be to test expected failing cases

Reply:
solved this by implementing the `not` operator